soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -92,6 +92,14 @@ config FLASH_SIZE
 	default $(dt_node_int_prop_int,$(DT_CHOSEN_FLASH_NODE),size,Kb) \
 		if $(DT_FLASH_HAS_SIZE_PROP)
 
+config FORCE_DEFAULT_FLEXRAM_FUSE
```

Comment:
Maybe add a comment here explaining why we need to keep the flexram at the boot time allocation when using ITCM or DTCM as flash?

Reply:
I actually think this might not be necessary, I just tested with and without it and the behavior is no different, I might remove it. Because, there is nothing even being used in the RAM during the time we configure the banks, since even the reset handler did not actually run yet and do the data copy / init etc